### ISSUE TITLE
Add a key to undo editing (and maybe other aspects)

### ISSUE THREAD

**Issue by neiljp** [zulip/zulip-terminal#576]
This would probably be just in the editor for now, useful for accidentally deleting words/lines/everything via shortcut keys for example, but could be extended to 'undo' exiting a compose box (related to drafts v0?) or maybe editing a message (with a confirmation popup?)

Since we don't use ctrl-z for suspend, this key could be used, unless we want to support suspend in future?

**Comment by kaustubh-nair** [comment]
@neiljp We already support a basic version of undo provided by `readline` which can be used to undo accidental deletions. (I believe the shortcut is `ctrl + _ ` ). We should probably add this to the help text, and even change the default shorcut key. This should make a good first issue

**Comment by neiljp** [comment]
@kaustubh-nair Right; one aspect from an enhancement to `urwid-readline` which we discussed with them was how we could expose their key-mapping to allow generation of part of the help menu automatically in this way, though this has not been attempted.
